libmattermost.c

```diff
@@ -3706,15 +3711,16 @@ mm_got_history_of_room(MattermostAccount *ma, JsonNode *node, gpointer user_data
 	gint i, len = json_array_get_length(order);
 
 	// do not show updates (such as reactions). only edits, new posts and deletes
-	for (i = len - 1; i >= 0; i--) {
-		const gchar *post_id = json_array_get_string_element(order, i);
-		JsonObject *post = json_object_get_object_member(posts, post_id);
-
-		const gint64 since = mm_get_channel_approximate_view_time(ma, channel);
-		if (json_object_get_int_member(post, "create_at") < since && json_object_get_int_member(post, "edit_at") < since && json_object_get_int_member(post, "delete_at") < since) {
-			json_array_remove_element(order, i);
-		}
-	}
+	// We don't need this now, as in a channel we are fetching last 60 messages where post "create_at" could be lesser than channel joined time "since"

```

Comment:
```suggestion
	// We don't need this now, as in a channel we are fetching last 60 messages where post "create_at" time could be lesser than channel joined time "since"

```